README.md

```diff
@@ -114,6 +116,16 @@ Running unit tests
 BOARD_SIZE=9 python3 -m unittest discover tests
 ```
 
+Automated Tests
+----------------
+
+Minigo uses the Kubernetes
+[Prow](https://github.com/kubernetes/test-infra/tree/master/prow) to
+automatically test PRs. You can see the status of our automated tests by
```

Comment:
Can you say in one sentence how it does this?  Like "Prow pulls commits into a hermetic environment managed by [where?] and runs our unittests, builds our containers, runs our integration tests, and checks our lint rules"..... or something :)

Reply:
Done